lib/background_worker.rb

```diff
@@ -1,30 +1,47 @@
-module BackgroundWorker
-  VERSION = '0.0.1'
-
-  require 'background_worker/base'
-  require 'background_worker/persistent_state'
-
-  def self.after_exception(&block)
-    if block
-      @@after_exception = block
-    else
-      @@after_exception if defined?(@@after_exception)
-    end
-  end
+require 'background_worker/version'
+require 'background_worker/uid'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.